gensrc/script/functions.py

```diff
@@ -569,6 +569,26 @@
 
     [70415, 'esquery', 'BOOLEAN', ['VARCHAR', 'VARCHAR'], 'ESFunctions::match'],
 
+    [70500, 'ifs', 'BOOLEAN', ['BOOLEAN', '...'], 'nullptr'],
```

Comment:
Why is this also needed if it will not execute in be?

Reply:
Fix:
1. rebase to main
2. remove invalid code and fix document format